pkgs/data/themes/rose-pine-gtk/default.nix

```
@@ -1,20 +1,20 @@
-{ stdenv
-, fetchFromGitHub
-, lib
-, gnome-themes-extra
-, gtk-engine-murrine
-, gtk_engines
+{
+  stdenv,
+  fetchFromGitHub,
+  lib,
+  gnome-themes-extra,
+  gtk-engine-murrine,
+  gtk_engines,
 }:
```

```suggestion
{ stdenvNoCC
, lib
, fetchFromGitHub
, gnome-themes-extra
, gtk-engine-murrine
, gtk_engines
, gtk
}:

```
Usually this is the formatting I would use here, suggested by `nixpkgs-fmt`.

Also use `stdenvNoCC`, because we do not need a complete compiler toolchain for this package.